src/components/display/DecDisplay.vue

```diff
@@ -40,14 +40,15 @@ export default {
       showCopyIcon: false,
 
       displayFormatOptions: ["decimalDegrees", "sexagesimalWithUnits", "sexagesimalPlain"],
-      displayFormatSelected: 0, // index of the list above
+      displayFormatSelected: window.localStorage.getItem('dec_display_format'), // index of the list above
```

Comment:
Does the code handle the case where nothing has been set to localStorage yet?

Reply:
Same question for RA too.

Reply:
I edited the displayFormat() function down below to set the display format to the first option if this returns null - it seemed to work when I opened it in a new browser!

Reply:
Oh I see, great! No need to change, but another alternative would be 
`displayFormatSelected: window.localStorage.getItem('dec_display_format') ?? <default value>`
which uses the nullish coalescing operator (cool name) to set a fallback if the first expression has a nullish value (null or undefined).

This is a new variant of the more familiar || operator, which returns the right side if the left is null, undefined, _or also_ evaluates to a falsy value such as 0, '', or an actual `false`. 

Just another technique to have in your toolbox.